design/read-only-registry.md

```diff
@@ -0,0 +1,174 @@
+# Support for Read-Only Registries
+
+## Goals
+
+- Configure kotsadm to use application images from a private registry
+- Support using read-only registry during initial installation
+- Support using read-only registry on application updates
+- Support using read-only registry when settings are modified on the Registry Settings page in Admin Console
+- Support read-only registries in airgap and on-line modes
+
+## Non Goals
+
+- Delivering images to the read-only registry.
+- Creating imageless airgap bundles.
+- Addressing existing inconsistencies and bugs in multi-app support.
```

Comment:
This is a bit of a strange one. I can't guess why this is included. Can you elaborate?

Reply:
There are some issues with registry settings in multi-app setups.  They are tied to application but also used globally for Admin Console itself.